### PR TITLE
feat (#minor): seaport added fulfillment types and deployed to explorer

### PR DESCRIPTION
**Pull request by smith-phil**
This deploys the seaport subgraph to [Explorer](https://thegraph.com/explorer/subgraph?id=9ZvDz8qtx3PrnaUw1duQ7EhgGphgQcgSHqhLzQXBa16u&view=Indexers). I've added the following internal entities/enums:

- _OrderFulfillment [entity]
  Acts as a lookup between a trade the order fulfillment type used to execute the trade

- _OrderFulfillmentMethod [enum]
  The fulfillment methods available in seaport, used by _OrderFulfillment

The type of order fulfillment is used to determine the [strategy for a trade](https://github.com/messari/subgraphs/blob/12df4ea82e0b836b902f079d188b54a0953ea12b/subgraphs/seaport/src/mapping.ts#L151).

### CODE REVIEW

**Comment by this-username-is-taken** [comment]
@smith-phil appreciate the change! Would you fix the lint issues and we can get it in?

File: subgraphs/seaport/src/helper.ts [hunk 1]
```diff
@@ -1,4 +1,32 @@
...
+  export const FULFILL_BASIC_ORDER = "FULFILL_BASIC_ORDER" 
+  export const FULFILL_ORDER = "FULFILL_ORDER" 
+  export const FULFILL_ADVANCED_ORDER = "FULFILL_ADVANCED_ORDER" 
+  export const FULFILL_AVAILABLE_ORDERS = "FULFILL_AVAILABLE_ORDERS"
+  export const FULFILL_AVAILABLE_ADVANCED_ORDERS = "FULFILL_AVAILABLE_ADVANCED_ORDERS"
+  export const MATCH_ORDERS = "MATCH_ORDERS" 
+  export const MATCH_ADVANCED_ORDERS = "MATCH_ADVANCED_ORDERS" 
+}
+
+export namespace MethodSignatureTypeStrings {
```
Comment (this-username-is-taken) [inline subgraphs/seaport/src/helper.ts:23]: is this being used anywhere?

Comment (smith-phil) [reply subgraphs/seaport/src/helper.ts:23]: @this-username-is-taken Sure, I'll get these fixed over the next week.

**Comment by smith-phil** [comment]
Once the merge is accepted and complete do you want me to update the subgraph details to point to this repo instead of mine, and add messari.io as the website?

**Review by this-username-is-taken — APPROVED** [review]

**Comment by this-username-is-taken** [comment]
@smith-phil also I'd love to learn more about what you are doing with the Opensea subgraph, and get any feedback about our subgraphs from you. We also have a Discord that I can add you to which can help with communication. My Discord handle is Vincent#8080

**Comment by smith-phil** [comment]
@this-username-is-taken just sent you a friend request, it's from EvilPhil#3513

**Pull request merged**